client/src/utils/KBar.tsx

```diff
@@ -63,6 +63,120 @@ export const KBar: React.FunctionComponent<PropsWithChildren> = ({ children }) =
         dispatch(fetchAllCharacters())
       },
     },
+    {
+      id: 'configuration',
+      name: 'Choose your configuration options...',
+      keywords: 'interface color dark light',
+      section: 'Configuration',
+    },
+    {
+      id: 'ledger',
+      name: 'Select your preferred ledger',
+      keywords: 'ledger',
+      section: '',
```

Comment:
Is section mandatory? If not I would remove it, if it is maybe we can put in something useful

Reply:
I will have to check if it is, but from the way it currently works, empty section puts that item in the root menu. And most of those items would need to have a specific section unique to them if we were to assign them values, which would propbably just clutter the menu and be of no real use